wifiphisher/common/extensions.py

```diff
@@ -379,9 +400,11 @@ def _send(self):
             for pkt in self._packets_to_send[self._current_channel] + \
                     self._packets_to_send["*"]:
                 try:
-                    logger.debug("Send pkt with addr1:%s addr3:%s subtype:%s in channel:%s",
-                                 pkt.addr1, pkt.addr2, pkt.subtype, self._current_channel)
-                    self._socket.send(pkt)
+                    if is_deauth_cont or not self._is_deauth_frame(pkt):
+                        logger.debug("Send pkt with A1:%s A2:%s subtype:%s in channel:%s",
+                                     pkt.addr1, pkt.addr2, pkt.subtype,
+                                     self._current_channel)
```

Comment:
This is not the right place for this kind of logging. It should be placed in death module instead.

Reply:
This is the final pkt enqueueing place and we can check all the frames cross the extension modules not just with the deauthing frames. How do you think?